### PR TITLE
Changelog for APIM - 3.18.0

### PR DESCRIPTION
**Pull request by graviteeio**
Generate changelog for version APIM - 3.18.0

### CODE REVIEW

File: CHANGELOG-v3.adoc [hunk 1]
```diff
@@ -5,6 +5,83 @@ For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim
...
+
+- API properties header title change https://github.com/gravitee-io/issues/issues/6065[#6065]
+
+*_Platform_*
+
+- Mutualize System proxy configuration https://github.com/gravitee-io/issues/issues/7739[#7739]
+
+ 
+ 
+
```
Comment (mouligno) [inline CHANGELOG-v3.adoc:84]:
```suggestion
== https://github.com/gravitee-io/issues/milestone/519?closed=1[APIM - 3.18.0 (2022-07-07)]

=== Bug fixes

*_General_*

- Merge 3.17.2 into master https://github.com/gravitee-io/issues/issues/7617[#7617]
- Retry Policy - cancel timeout response, manage lastResponse counter and tests https://github.com/gravitee-io/issues/issues/7747[#7747]

*_Management_*

- DCR providers should be scoped by org https://github.com/gravitee-io/issues/issues/6604[#6604]
- Not_equals alert filter displays an empty list https://github.com/gravitee-io/issues/issues/7489[#7489]
- One shot upgraders run on each APIM startup with cockpit https://github.com/gravitee-io/issues/issues/7450[#7450]
- Wrong example when generating Personal Access Token https://github.com/gravitee-io/issues/issues/5271[#5271]
- Remove the horizontal scroll bar in the markdown creation page https://github.com/gravitee-io/issues/issues/5119[#5119]
- OpenApi files are never updated https://github.com/gravitee-io/issues/issues/7631[#7631]
- Gateway Testing SDK - Data Logging Masking Policy https://github.com/gravitee-io/issues/issues/7758[#7758]

=== Features

*_Console_*


*_General_*

- Promote API Designer https://github.com/gravitee-io/issues/issues/7645[#7645]
- Support of RHEL8 https://github.com/gravitee-io/issues/issues/7208[#7208]
- Write documentation for Circuit Breaker policy https://github.com/gravitee-io/issues/issues/7756[#7756]
- Add Pendo analytics tool https://github.com/gravitee-io/issues/issues/7781[#7781]

*_Management_*

- Partial update - PATCH method on Import API https://github.com/gravitee-io/issues/issues/7443[#7443]
- Add page to display organization Audit https://github.com/gravitee-io/issues/issues/7536[#7536]

*_Policy_*

- Define headers based on the request or on the response payload https://github.com/gravitee-io/issues/issues/7359[#7359]

=== Improvements

*_General_*

- Add Conditional icon in legend https://github.com/gravitee-io/issues/issues/7457[#7457]
- Icons not rendering with custom nginx configuration https://github.com/gravitee-io/issues/issues/7569[#7569]

*_Management_*

- API properties header title change https://github.com/gravitee-io/issues/issues/6065[#6065]

*_Platform_*

- Mutualize System proxy configuration https://github.com/gravitee-io/issues/issues/7739[#7739]



```

**Review by mouligno — COMMENTED** [review]
@gravitee-io/apim what do you think ?

File: CHANGELOG-v3.adoc (hunk 1 above)
Comment (phiz71) [reply CHANGELOG-v3.adoc:84]: I would have change some lines:

```suggestion
== https://github.com/gravitee-io/issues/milestone/519?closed=1[APIM - 3.18.0 (2022-07-07)]

=== Bug fixes
*_Console_*

- Remove the horizontal scroll bar in the markdown creation page https://github.com/gravitee-io/issues/issues/5119[#5119]
- Wrong example when generating Personal Access Token https://github.com/gravitee-io/issues/issues/5271[#5271]
- Not_equals alert filter displays an empty list https://github.com/gravitee-io/issues/issues/7489[#7489]
- Icons not rendering with custom nginx configuration https://github.com/gravitee-io/issues/issues/7569[#7569]

*_General_*

- Merge 3.17.2 into master https://github.com/gravitee-io/issues/issues/7617[#7617]

*_Management_*

- DCR providers should be scoped by org https://github.com/gravitee-io/issues/issues/6604[#6604]
- One shot upgraders run on each APIM startup with cockpit https://github.com/gravitee-io/issues/issues/7450[#7450]
- OpenApi files are never updated https://github.com/gravitee-io/issues/issues/7631[#7631]

*_Policies_*

- Retry Policy: cancel timeout response, manage lastResponse counter and tests https://github.com/gravitee-io/issues/issues/7747[#7747]
- Data Logging Masking: fix some bugs https://github.com/gravitee-io/issues/issues/7758[#7758]

=== Features

*_Console_*

- Promote API Designer https://github.com/gravitee-io/issues/issues/7645[#7645]
- Add Pendo analytics tool https://github.com/gravitee-io/issues/issues/7781[#7781]

*_General_*

- Check Support of RHEL8 https://github.com/gravitee-io/issues/issues/7208[#7208]

*_Management_*

- Partial update - PATCH method on Import API https://github.com/gravitee-io/issues/issues/7443[#7443]
- Add page to display organization Audit https://github.com/gravitee-io/issues/issues/7536[#7536]

*_Policies_*

- Transform-Header: Define headers based on the request or on the response payload https://github.com/gravitee-io/issues/issues/7359[#7359]
- Circuit Breaker: Write documentation for policy https://github.com/gravitee-io/issues/issues/7756[#7756]

=== Improvements

*_Console_*

- API properties header title change https://github.com/gravitee-io/issues/issues/6065[#6065]
- Add Conditional icon in legend https://github.com/gravitee-io/issues/issues/7457[#7457]

*_General_*

- Mutualize System proxy configuration https://github.com/gravitee-io/issues/issues/7739[#7739]



```

Comment (marcambier) [reply CHANGELOG-v3.adoc:84]: :+1:

**Pull request merged**